pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -52,6 +52,16 @@ type VPCNetworkConfigurationSpec struct {
 type VPCNetworkConfigurationStatus struct {
 	// Conditions describes current state of VPCNetworkConfiguration.
 	Conditions []Condition `json:"conditions"`
+	// LBSubnets describes lb subnets info, now it is needed for AKO.
+	LBSubnets []LBSubnetInfo `json:"lbSubnets,omitempty"`
+}
+
+// LBSubnetInfo defines lb subnet info.
+type LBSubnetInfo struct {
+	// NSX PolicyPath for the load balancer Subnet.
+	LBSubnetPath string `json:"lbSubnetPath,omitempty"`
+	// CIDR for the load balancer Subnet.
+	LBSubnetCIDR string `json:"lbSubnetCIDR,omitempty"`
```

Comment:
Does AKO need this one? This should be in NetworkInfo for Namespace users to know the CIDR and define Sec Policies to allow LB traffic. And as we talked earlier, we should change it to be more generic to support NSX LB SNAT IP. So, consider something like `LoadBalancerIPAddresses`.

Reply:
Reminder.

Reply:
I discussed with Abishek before and he mentioned they only needs AVI SE subnet path, so removed this field for now, I will double check with him.

Reply:
Abishek said they do not need the AVI SE Subnet CIDR, I will remove it for now.